assets/css/admin.scss

```diff
@@ -1129,3 +1129,26 @@ h2 .nav-tab-active, nav .nav-tab-active {
 .jm-logo__wrapper:hover .jm-logo__letters {
 	transform: rotateY(90deg);
 }
+
+/**
+ * Adjust upsells styles
+ */
+.wpjm-addon-upsell__badge {
+	display: inline-block;
+	vertical-align: middle;
+	box-sizing: border-box;
+	margin: 1px 0 -1px 2px;
+	padding: 0 3px; /* Reduced padding */
```

Comment:
I don't think the comments are needed for these :)

Reply:
Fair. Removed in d33d5e54e108b6818332e74c148ae3bc23b00cc5. Thanks